garrysmod/gamemodes/terrortown/gamemode/traitor_state.lua

```diff
@@ -179,6 +179,7 @@ local function force_spectate(ply, cmd, arg)
       end
    end
 end
-concommand.Add("ttt_spectate", force_spectate)
-
+net.Receive("ttt_spectate", function(l,pl)

```

Comment:
register the network string using [](http://wiki.garrysmod.com/page/util/AddNetworkString)